file_paths.md

```diff
@@ -292,6 +294,7 @@ create a `puppet` user or group.
             puppet *                          # ruby bin wrapper
         lib *
             puppet.rb *
+        locale *
```

Comment:
From the translation side, I specify the location of the PO file using the file path like this:

`[transifex-test-3.japo]
source_file = locales/ja_JP/ja.po
source_lang = en
type = PO
lang_map = ja: ja_JP
file_filter = locales/ja_JP/ja.po` 

I don't think it matters on the Transifex side where the PO file is located, as long as we provide the correct file path.